decidim-pages/config/locales/es.yml

```diff
@@ -0,0 +1,5 @@
+---
+es:
```

Comment:
Nothing happens if I include the translations here. It actually gets imported into crowdin to be verified later.